tests/foreman/api/test_capsulecontent.py

```diff
@@ -868,18 +868,18 @@ def test_positive_sync_kickstart_repo(
         module_capsule_configured.wait_for_sync(start_time=timestamp)
         cvv = cvv.read()
         assert len(cvv.environment) == 2
-
         # Check for kickstart content on SAT and CAPS
         tail = (
             f'rhel/server/7/{REPOS["kickstart"][distro]["version"]}/x86_64/kickstart'
             if distro == 'rhel7'
+            else f'{distro.split("_")[0]}/{REPOS["kickstart"][distro]["version"]}/beta/x86_64/baseos/kickstart'
+            if 'beta' in distro  # for future beta rhel distros
             else f'{distro.split("_")[0]}/{REPOS["kickstart"][distro]["version"]}/x86_64/baseos/kickstart'  # noqa:E501
         )
```

Comment:
now that you're injecting the two additional lines (and additional condition), i'd recommend the same as earlier.